build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
         api("com.google.protobuf:protoc:${protobufVersion}")
 
         api("org.slf4j:slf4j-api:${slf4jVersion}")
+        api("org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}")
```

Comment:
Books-pages common can't compile with this version of bom. Also, this library could potentially break compatibility with both log4j formats. Additional tests required